retailer-oas/paths/timeslots/timeslots.yaml

```diff
@@ -39,22 +32,64 @@ get:
       example: 1
       schema:
         type: number
+    - name: service
```

Comment:
question: what is this about?

Reply:
A prep for what will come in the future. The possibility of filtering timeslots/PLs that offer different services if needed.